sources/packages/web/src/types/contracts/aest/roles.ts

```diff
@@ -5,6 +5,7 @@ export enum Role {
   /**
    * AEST roles.
    */
+  AESTAdmin = "aest-admin",
```

Comment:
Please review once the below one is changed
https://github.com/bcgov/SIMS/pull/3042/files#r1541629352